### ISSUE TITLE
Can't use xdebug - ObjectRegistry __debugInfo()

### ISSUE THREAD

**Issue by QuanticPotato** [cakephp/cakephp#7604]
Hi, I'm trying to use xdebug with CakePHP (and vim as remote).

It works fine with no breakpoints, but crash if I set some (for example in AppController::beforeFilter()).

The PHP error is : 
```
Notice (8): Undefined index: _loaded [CORE/src/Core/ObjectRegistry.php, line 319]
Code Context

    {
        $properties = get_object_vars($this);
        $properties['_loaded'] = array_keys($properties['_loaded']);

Cake\Core\ObjectRegistry::__debugInfo() - CORE/src/Core/ObjectRegistry.php, line 319
App\Controller\Admin\Cards\LevelsController::beforeFilter() - APP/Controller/Admin/Cards/LevelsController.php, line 35
Cake\Event\EventManager::_callListener() - CORE/src/Event/EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE/src/Event/EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE/src/Event/EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE/src/Controller/Controller.php, line 491
Cake\Routing\Dispatcher::_invoke() - CORE/src/Routing/Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE/src/Routing/Dispatcher.php, line 87
[main] - ROOT/webroot/index.php, line 13
```

First of all, I don't understand why this method (__debugInfo) is called ?

Moreover, after this error apache crashes.

Thanks for your help

**Comment by markstory** [comment]
Xdebug is calling `__debugInfo()` as part of the PHP5.6 `var_dump()` improvements which allow userland code to define the output of var_dump. I'll fix up the notice error.

**Comment by markstory** [comment]
Fixed with #7605
